appveyor.yml

```diff
@@ -52,7 +63,17 @@ before_build:
   - set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
   - if %platform%==msvc call "C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\vcvarsall.bat"
   - if %platform%==msvc cmake -G "NMake Makefiles" -DCMAKE_INSTALL_PREFIX=%P%
-  - if %platform%==vs cmake -G "Visual Studio 14 2015 Win64" -DCMAKE_INSTALL_PREFIX=%P%
+  - if %platform%==vs2005_64 cmake -G "Visual Studio 8 2005" -DCMAKE_INSTALL_PREFIX=%P%
```

Comment:
This adds more variations for compilation, but not running the unit tests. Should they also run the build unit tests?

Reply:
Aren't the unit tests run just for the code built by the selected compiler?
At least I saw some tests running.

But there are some extra lines:

if %platform%==vs msbuild "ALL_BUILD.vcxproj"

if %platform%==vs ctest --extra-verbose -C Release

I will check what difference they really make. AppVeyor seemed pretty complete without them ...

Reply:
I believe the ctest lines are what run the tests. When I looked at the AppVeyor output for the new platforms the unit tests were not being run.